benchmarking-tests/benchmark_utils.py

```diff
@@ -95,12 +98,12 @@ def run_gather_metrics(args):
                 percent_complete = round((num_transactions_received / num_transactions) * 100)
 
                 if elapsed_time != 0:
-                    transactions_per_second = round(num_transactions_received / elapsed_time)
+                    transactions_per_second = round(num_transactions_received /
+                                                    elapsed_time)
                 else:
                     transactions_per_second = float('nan')
 
-                logger.info('\t{:<20} {:<20} {:<20} {:<20}'.format(time_now, block_num_transactions,
-                                                                   transactions_per_second, percent_complete))
+                logger.info('\t{:<20} {:<20} {:<20} {:<20}'.format(time_now, block_num_transactions, transactions_per_second, percent_complete))
```

Comment:
As you may know PEP 8 is somewhat tolerant on the line length. The strict limit is 79, and we encourage developers to follow this limit whenever possible, but we nevertheless tolerate up to 119 characters,, as it may sometimes be preferable to have longer lines. That being said, the above line is 144 characters long, which is a bit too long according to our [style guide](https://github.com/bigchaindb/bigchaindb/blob/master/PYTHON_STYLE_GUIDE.md#maximum-line-length). 

I would suggest something like:

```python
                logger.info('\t{:<20} {:<20} {:<20} {:<20}'.format(
                    time_now,
                    block_num_transactions,
                    transactions_per_second,
                    percent_complete,
                ))
```